src/context.ts

```diff
@@ -179,7 +181,23 @@ export class NuxtIconModuleContext {
     const { getIconData } = await import('@iconify/utils')
     const { loadCollectionFromFS } = await import('@iconify/utils/lib/loader/fs')
 
+    const failed: string[] = []
+    let count = 0
+
+    const customCollectionNames = new Set(customCollections.map(c => c.prefix))
     const collections = new Map<string, IconifyJSON>()
+
+    function loadCollection(prefix: string) {
+      if (customCollectionNames.has(prefix)) {
+        const collection = customCollections.find(c => c.prefix === prefix)
```

Comment:
`.find` should be alright here. We enter this path only once per custom collection. Optimizing lookup seems like an overkill for the sensible length of `customCollections`.